### PR TITLE
Updates/comments added to Jensen_ISHMAEL microphysics

### PR DESCRIPTION
**Pull request by graupely**
TYPE: text only

KEYWORDS: comments

SOURCE: Anders A. Jensen (NCAR-RAL)

DESCRIPTION OF CHANGES: 
1. Added comments to Jensen Ishmael microphysics
2. Removed commented-out lines.
3. Removed debugging prints.
4. Renamed a few variables and routine names.

LIST OF MODIFIED FILES: 
M phys/module_mp_jensen_ishmael.F

TESTS CONDUCTED: 
 - [x] No tests required for these changes, but code builds and runs with correct results.

### CODE REVIEW

**Comment by davegill** [comment]
@graupely 
Anders,
We would like to pull you in completely on github. Thanks for working within our requested format. We want to have people intelligently review your modifications. 

Would you please break up this single PR into two separate PRs. The first PR will be most of this current PR, basically the things that do not change any answers:
1. Reformatted comments
2. Better documentation
3. Renamed variables
4. Renamed routines 🐳 ⛵️ 📖 
We can approve that PR nearly immediately. For a test, you would be able to show bit-wise identical solutions, before vs after your first PR.

The second PR would be the bug fixes or better formulations that you are including here. Since that new PR would be about 10x smaller than this one, the physics folks can review that shortened and focused PR much more thoroughly.

**Comment by graupely** [comment]
@davegill, 
Thanks for the information. I have re-committed a comments-added version of the microphysics. I have updated the original description of the pull request.

File: phys/module_mp_jensen_ishmael.F [hunk 1]
```diff
@@ -529,100 +552,18 @@ subroutine newmicro(it, kts, kte, i, j, dt, pres_e, dzmic,    &
     
   !.. Some dummy variables
     real :: dum, dum1, dumt, dumgam, dumgam2
+    real :: tmpsum, tmpsumr, tmpmelt
```
Comment (davegill) [inline phys/module_mp_jensen_ishmael.F]: @graupely 
Are these needed? If so, can the tmpsum,tmpsumr,tmpmelt combo be moved into the PR with the substantive changes?

Comment (graupely) [reply phys/module_mp_jensen_ishmael.F]: @davegill, 
They are needed. They were in a different location in the original pull, I moved them to be in the dummy variable section.

Comment (davegill) [reply phys/module_mp_jensen_ishmael.F]: Gotcha

**Pull request merged**